crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -52,6 +54,22 @@ def knife_files
           file_path = new_file
         end
 
+        if file.basename.to_s =~ /^bc-network-(.*)\.json$/
+          json = JSON.load(file.read)
+          Tempfile.open("network") do |content|
+            content.write JSON.pretty_generate(json["attributes"])
+            cmd = [
+              "sudo",
+              "echo",
+              content.read,
+              ">",
+              "/etc/crowbar/network.json"
```

Comment:
Why not copy the file? Sounds a bit safer than redirecting the output of an `echo`.